lib/Email/Valid.pm

```diff
@@ -157,6 +158,15 @@ sub _net_dns_query {
     }
   }
 
+  # Chceck for A record for $host
+  my @a_rrs = Net::DNS::rr($Resolver, $host, 'A');
```

Comment:
Although if your local SMTP server that you are using can't handle IPV6, then testing both A and AAAA might not be good if the domain resolves to only an AAAA record.

Maybe AAAA should be an optional parameter to enable IPV6 checks.

By default, Net::DNS searches default to "A IN"